README.md

```diff
@@ -55,6 +55,26 @@ end
 
 See also: [test/reline/yamatanooroti/multiline_repl](https://github.com/ruby/reline/blob/master/test/reline/yamatanooroti/multiline_repl)
 
+## Contributing
+
+Bug reports and pull requests are welcome on GitHub at https://github.com/ruby/reline.
+
+### Run tests
+
+```bash
+WITH_VTERM=1 bundle install
```

Comment:
I think we need to include how to properly build and install `vterm` locally too. I have installed `libvterm` through homebrew but installing the `vterm` gem still got me this error:

```
compiling vterm.c
vterm.c:10:10: fatal error: 'vterm.h' file not found
#include "vterm.h"
         ^~~~~~~~~
1 error generated.
make: *** [vterm.o] Error 1
```

Reply:
I had the same problem with my new M2 mac, (not in my old intel mac)
Can you try this? https://github.com/ruby/vterm-gem/pull/4

Reply:
That works, thank you!